src/views/Card/CardGroup.js

```diff
@@ -0,0 +1,63 @@
+import cx from 'classnames'
+import React, { PropTypes } from 'react'
+
+import {
+  customPropTypes,
+  getUnhandledProps,
+  META,
+  numberToWord,
+  SUI,
+  useKeyOnly,
+} from '../../lib'
+import Card from './Card'
+
+function CardGroup(props) {
+  const { className, children, doubling, items, itemsPerRow, stackable } = props
+  const classes = cx('ui',
+    numberToWord(itemsPerRow),
+    useKeyOnly(doubling, 'doubling'),
+    useKeyOnly(stackable, 'stackable'),
+    className,
+    'cards',
+  )
+  const rest = getUnhandledProps(CardGroup, props)
+
+  const content = !items ? children : items.map(item => {
+    const key = item.key || [item.header, item.description].join('-')
+    return <Card key={key} {...item} />
+  })
+
+  return <div {...rest} className={classes}>{content}</div>
+}
+
+CardGroup._meta = {
+  name: 'CardGroup',
+  parent: 'Card',
+  props: {
+    itemsPerRow: SUI.WIDTHS,
+  },
+  type: META.TYPES.VIEW,
+}
+
+CardGroup.propTypes = {
+  className: PropTypes.string,
+  children: customPropTypes.every([
+    customPropTypes.disallow(['items']),
+    PropTypes.node,
+  ]),
+  doubling: PropTypes.bool,
+  items: customPropTypes.every([
+    customPropTypes.disallow(['description', 'header']),
+    PropTypes.arrayOf(PropTypes.shape({
+      description: PropTypes.node,
+      meta: PropTypes.node,
+      key: PropTypes.string,
+      header: PropTypes.node,
+    })),
+  ]),
+  itemsPerRow: PropTypes.oneOf(CardGroup._meta.props.width),
+  stackable: PropTypes.bool,
```

Comment:
All `propTypes` need a doc block.  This is what shows up on the doc site to document the props.  We use the SUI doc descriptions when available.  Here's a couple examples:

```jsx
/** A group of cards can double its column width for mobile */
doubling: PropTypes.bool,
```
>http://semantic-ui.com/views/card.html#doubling

```jsx
/** A group of cards can automatically stack rows to a single columns on mobile devices */
stackable: PropTypes.bool,
```
>http://semantic-ui.com/views/card.html#stackable